[2025-02-02T15:37:56.886+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T15:37:56.912+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-02-02T15:37:50.956921+00:00 [queued]>
[2025-02-02T15:37:56.925+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-02-02T15:37:50.956921+00:00 [queued]>
[2025-02-02T15:37:56.926+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-02T15:37:56.942+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_file_in_directory> on 2025-02-02 15:37:50.956921+00:00
[2025-02-02T15:37:56.950+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1338) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T15:37:56.952+0000] {standard_task_runner.py:72} INFO - Started process 1340 to run task
[2025-02-02T15:37:56.953+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_file_in_directory', 'manual__2025-02-02T15:37:50.956921+00:00', '--job-id', '172', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmp4to4usy1']
[2025-02-02T15:37:56.955+0000] {standard_task_runner.py:105} INFO - Job 172: Subtask check_file_in_directory
[2025-02-02T15:37:57.191+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-02-02T15:37:50.956921+00:00 [running]> on host 003b7fbea265
[2025-02-02T15:37:57.287+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_file_in_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T15:37:50.956921+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T15:37:50.956921+00:00'
[2025-02-02T15:37:57.288+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-02T15:37:57.289+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-02T15:37:57.290+0000] {logging_mixin.py:190} INFO - Current task name:check_file_in_directory state:running start_date:2025-02-02 15:37:56.913037+00:00
[2025-02-02T15:37:57.291+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-02T15:37:57.292+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T15:37:57.293+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-02T15:37:57.298+0000] {python.py:240} INFO - Done. Returned value was: verify_file_reliability
[2025-02-02T15:37:57.298+0000] {branch.py:38} INFO - Branch into verify_file_reliability
[2025-02-02T15:37:57.299+0000] {skipmixin.py:230} INFO - Following branch ('verify_file_reliability',)
[2025-02-02T15:37:57.315+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-02T15:37:57.348+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T15:37:57.349+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_file_in_directory, run_id=manual__2025-02-02T15:37:50.956921+00:00, execution_date=20250202T153750, start_date=20250202T153756, end_date=20250202T153757
[2025-02-02T15:37:57.360+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-02T15:37:57.361+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-02T15:37:57.362+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-02 15:37:50.971241+00:00
[2025-02-02T15:37:57.363+0000] {logging_mixin.py:190} INFO - Task hostname:003b7fbea265 operator:BranchPythonOperator
[2025-02-02T15:37:57.411+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T15:37:57.603+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T15:37:57.607+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
